src/ansys/geometry/core/math/misc.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+"""Provides auxiliary math functions for the PyAnsys Geometry library."""
+
+from typing import Tuple, Union
+
+from beartype import beartype as check_input_types
+import numpy as np
+
+from ansys.geometry.core.typing import Real
+
+
+@check_input_types
+def get_two_circle_intersections(
+    x0: Real, y0: Real, r0: Real, x1: Real, y1: Real, r1: Real
+) -> Union[Tuple[Real, Real, Real, Real], None]:
+    """
+    Get the intersection points of two circles.
+
+    Parameters
+    ----------
+    x0 : Real
+        x-coordinate of the first circle.
+    y0 : Real
+        y-coordinate of the first circle.
+    r0 : Real
+        Radius of the first circle.
+    x1 : Real
+        x-coordinate of the second circle.
+    y1 : Real
+        y-coordinate of the second circle.
```

Comment:
```suggestion
    x0 : Real
        x coordinate of the first circle.
    y0 : Real
        y coordinate of the first circle.
    r0 : Real
        Radius of the first circle.
    x1 : Real
        x coordinate of the second circle.
    y1 : Real
        y coordinate of the second circle.
```
No hyphen in x coordinate because it is an adjective and noun. If it were x-axis coordinate, then the compound adjective would be hyphenated.